harness/determined/common/experimental/trial.py

```diff
@@ -133,6 +140,7 @@ def __init__(self, trial_id: int, session: api.Session):
         self._session = session
 
         self.hparams: Optional[Dict[str, Any]] = None
+        self.summary: Optional[Dict[str, Any]] = None
```

Comment:
Why `summary` instead of `summary_metrics`? All else being equal, I've got a prejudice to preserve names across the various systems.

Reply:
you've got a good point. changed.

Reply:
oops, didn't push the commit before i responded to this.